README.md

```diff
@@ -28,8 +28,10 @@ See aces/Loris README.md for further information and Loris installation informat
 
 3. Run installer to install MINC & DICOM toolkits, Perl libraries, configure environment, and setup directories:
 
+   ```bash 
    cd /data/$projectname/bin/mri/
-   ```bash imaging_install.sh```
```

Comment:
Can you please make sure that it is explicitly running bash? We have had problems in the past if bash is not used.